components-core/src/main/java/com/adyen/checkout/components/util/PaymentMethodTypes.kt

```diff
@@ -39,6 +39,7 @@ object PaymentMethodTypes {
     const val ONLINE_BANKING_PL = "onlineBanking_PL"
     const val ONLINE_BANKING_SK = "onlineBanking_SK"
     const val PAY_BY_BANK = "paybybank"
+    const val PAY_NOW = "paynow"
```

Comment:
I believe `PAY_NOW` behaves like `PIX`, so it should be somewhere else in this file, check out `PIX`

Reply:
This is still not fixed